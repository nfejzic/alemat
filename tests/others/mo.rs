use alemat::{elements::Operator, MathMl};

#[test]
fn operator_product() {
    let out = MathMl::with_content(Operator::prod()).render();

    crate::snap_test!(out, name: "operator_product");
}

#[test]
fn operator_iff() {
    let out = MathMl::with_content(Operator::iff()).render();

    crate::snap_test!(out, name: "operator_iff");
}
