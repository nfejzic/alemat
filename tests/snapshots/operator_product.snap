---
source: tests/others/mo.rs
expression: out
---
<math><mo>∏</mo></math>
